NEWS.md

```diff
@@ -1,3 +1,9 @@
+# Moonlight2R 0.99.3
+
+## Summary
+
+* added tests with testthat

```

Comment:
I see you also changed some of the example data/available data, can you add a note here?

Reply:
added